packages/panzer/disc-fe/src/Panzer_DOFManagerFactory.cpp

```diff
@@ -40,13 +40,116 @@
 // ***********************************************************************
 // @HEADER
 
-#include "PanzerDiscFE_config.hpp"
+#ifndef PANZER_DOF_MANAGER_FACTORY_IMPL_HPP
+#define PANZER_DOF_MANAGER_FACTORY_IMPL_HPP
 
-#include "Panzer_DOFManagerFactory_decl.hpp"
-#include "Panzer_DOFManagerFactory_impl.hpp"
+#include "Panzer_DOFManager.hpp"
+#include "Panzer_DOFManagerFactory.hpp"
+#include "Panzer_IntrepidFieldPattern.hpp"
+#include "Panzer_IntrepidFieldPattern.hpp"
 
-template class panzer::DOFManagerFactory<int,int>;
+#include "Panzer_GlobalIndexer_Utilities.hpp"
+
+namespace panzer {
+
+Teuchos::RCP<panzer::GlobalIndexer> 
+DOFManagerFactory::buildGlobalIndexer(const Teuchos::RCP<const Teuchos::OpaqueWrapper<MPI_Comm> > & mpiComm,
+                                            const std::vector<Teuchos::RCP<panzer::PhysicsBlock> > & physicsBlocks,
+                                            const Teuchos::RCP<ConnManager> & connMngr,
+                                            const std::string & fieldOrder) const
+{
+  PANZER_FUNC_TIME_MONITOR_DIFF("panzer::DOFManagerFactory::buildUnqueGlobalIndexer",BUGI);
```

Comment:
- Unique

Reply:
"- Unique", there is another use of Unique do this further down

Reply:
Good catch. "Unique" has been purged! My script missed it due to mispelling.